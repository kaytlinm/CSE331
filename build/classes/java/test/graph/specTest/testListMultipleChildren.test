#List Multiple Children Test File
#Tests that a graph can be created, and a set of nodes and edges can be added in the formation of a tree (one parent node
#with multiple children nodes) can print out multiple children.

CreateGraph graph1
AddNode graph1 a
AddNode graph1 b
AddNode graph1 c
AddNode graph1 d

AddEdge graph1 a b edgeB
AddEdge graph1 a c edgeC
AddEdge graph1 a d edgeD

ListChildren graph1 a