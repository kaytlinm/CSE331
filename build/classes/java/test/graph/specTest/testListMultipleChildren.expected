#List Multiple Children Test File
#Tests that a graph can be created, and a set of nodes and edges can be added in the formation of a tree (one parent node
#with multiple children nodes) can print out multiple children.

created graph graph1
added node a to graph1
added node b to graph1
added node c to graph1
added node d to graph1

added edge edgeB from a to b in graph1
added edge edgeC from a to c in graph1
added edge edgeD from a to d in graph1

the children of a in graph1 are: b(edgeB) c(edgeC) d(edgeD)
